aspnetcore/tutorials/signalr-typescript-webpack.md

```diff
@@ -50,26 +50,42 @@ In this tutorial, you learn how to:
 
 Configure Visual Studio to look for npm in the *PATH* environment variable. By default, Visual Studio uses the version of npm found in its installation directory. Follow these instructions in Visual Studio:
 
+1. Launch Visual Studio 2019. At the start window, select **Continue without code**.
 1. Navigate to **Tools** > **Options** > **Projects and Solutions** > **Web Package Management** > **External Web Tools**.
-1. Select the *$(PATH)* entry from the list. Click the up arrow to move the entry to the second position in the list.
+1. Select the *$(PATH)* entry from the list. Click the up arrow to move the entry to the second position in the list, and select **OK**.
 
     ![Visual Studio Configuration](signalr-typescript-webpack/_static/signalr-configure-path-visual-studio.png)
 
 Visual Studio configuration is completed. It's time to create the project.
 
-1. Use the **File** > **New** > **Project** menu option and choose the **ASP.NET Core Web Application** template.
+1. Use the **File** > **New** > **Project** menu option and choose the **ASP.NET Core Web Application** template. Select **Next**.
 1. Name the project *SignalRWebPack*, and select **Create**.
 1. Select *.NET Core* from the target framework drop-down, and select *ASP.NET Core 3.0* from the framework selector drop-down. Select the **Empty** template, and select **Create**.
 
+Add the **Microsoft.TypeScript.MSBuild** package to your project:
```

Comment:
```suggestion
Add the `Microsoft.TypeScript.MSBuild` package to your project:
```

As a rule of thumb, code-fence package names to avoid localization.